spec/features/open_now_spec.rb

```diff
@@ -1,20 +1,70 @@
 require 'rails_helper'
 
-describe 'locations open now' do
-  context 'when one library closed and one open' do
+describe 'locations open now', type: :feature do
+  let(:now) { Time.now }
+  let(:tomorrow) { now + 1.day }
+  let(:fourAM) { Time.local(tomorrow.year, tomorrow.month, tomorrow.day , 4, 0, 0) }
```

Comment:
This should be Time.zone.local. Might be where those travis errors are coming from.